packages/components/breadcrumbs/src/breadcrumbs.ts

```diff
@@ -126,30 +127,37 @@ export class Breadcrumbs extends ScopedElementsMixin(LitElement) {
       ${this.noHome
         ? nothing
         : html`
-            <a href=${this.homeUrl}><sl-icon name="house"></sl-icon>${isMobile() ? '' : msg('Home')}</a>
-            <sl-icon name="chevron-right"></sl-icon>
+            <a href=${this.homeUrl}><sl-icon name="home-blank"></sl-icon>${isMobile() ? '' : msg('Home')}</a>
+            <sl-icon name="slash-forward"></sl-icon>
           `}
       ${this.breadcrumbs.length > this.collapseThreshold
         ? html`
-            <sl-menu-button fill="link">
-              <sl-icon name="ellipsis" slot="button"></sl-icon>
+            <sl-button fill="link" id="button" @click=${this.#onClick} icon-only aria-label=${msg('More breadcrumbs')}>
+              <sl-icon name="ellipsis"></sl-icon>
+            </sl-button>
+            <sl-popover anchor="button">
```

Comment:
There are some inconsistencies with Figma in the overflow version?

https://www.figma.com/file/1oIVcD9VMbOtLXYp48ehsA/Sanoma-Learning-Light-mode?type=design&node-id=6007-29995&mode=dev


Figma:
<img width="986" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/777e67c9-8f40-403c-9236-f3c8df6a4ab4">


Storybook:
<img width="404" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/e8779b68-1a4b-4de0-adfc-44a3b96fafce">

Reply:
I think this is a case where Figma needs to be updated with Storybook :)